### PR TITLE
update MariaDB manually with CLI (bsc#1132852, SOC-9022)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

**Comment by asettle** [comment]
Same comments on https://github.com/SUSE-Cloud/doc-cloud/pull/1055

File: xml/depl_maintenance.xml [hunk 1]
```diff
@@ -1816,14 +1816,13 @@ done</screen>
    10.2.22 Release Notes - Notable Changes</link>.
   </para>
   <para>
-   Update MariaDB with the following procedure:
+   Using the Pacemaker GUI, update MariaDB with the following procedure:
```
Comment (dmpop) [inline xml/depl_maintenance.xml]: There is an entity for Pacemaker.

Comment (csymons-suse) [reply xml/depl_maintenance.xml]: No entity for Pacemaker in Cloud 7 version of entity-decl.ent

Comment (asettle) [reply xml/depl_maintenance.xml]: That is true. Even so, it's literally "pacemaker": https://github.com/SUSE-Cloud/doc-cloud/blob/maintenance/cloud_9/xml/entity-decl.ent#L115

We could add it into cloud 7 but I imagine we're not going to be updating Cloud 7 that often anyway

**Pull request merged**